### ISSUE TITLE
test

### ISSUE THREAD

**Issue by AmberHen** [AlamiMejjati/plop#16]
Excuse me, I want to know what is the order of the images after the test? Is it random input and output?

**Comment by ayclove** [comment]
Hi. I have the same question. It looks like the input order is randomly shuffled. The order is not as same as the test.csv. Is there any option to make the input order as same as the .csv? Thanks in advance.

**Comment by xiaoguitoucode** [comment]
I have the same question. I add the parameter in tf.train.string_input_producer :[shuffle=False],but it doesnot work.(after the first 34 pictures, the order of images is still random. )
